weaviate/schema/validate_schema.py

```diff
@@ -97,9 +97,10 @@ def check_class(class_definition: dict) -> None:
         ]:
             _check_key_type(key, class_definition[key], dict)
         if key in ["properties"]:
-            _check_key_type(key, class_definition[key], list)
+            if (properties := class_definition[key]) is not None:
+                _check_key_type(key, properties, list)
 
-    if "properties" in class_definition:
+    if "properties" in class_definition and class_definition["properties"] is not None:
         for class_property in class_definition["properties"]:
             check_property(class_property)
```

Comment:
I thought that we should remove the complete thing - we had problems with that in the past for example new keys appear in the schema and then old versions suddenly do not work anymore

Reply:
it is still in the VCS, so we do not really need to keep it?